CI/sort_langfile_entries.sh

```diff
@@ -21,9 +32,26 @@ do
         fi
     done
 
-    echo  "sorting entries in ${file}";
+    #echo  "sorting entries in ${file}";
```

Comment:
We don't need to keep this line, we have the GIT history :-).

Reply:
Fixed.